test-suite/js/src/test/scala/org/scalajs/testsuite/typedarray/TypedArrayTest.scala

```diff
@@ -279,6 +319,11 @@ class Uint8ArrayTest extends TypedArrayTest[Short, Uint8Array] {
 object Uint8ClampedArrayTest extends Requires.TypedArray
 
 class Uint8ClampedArrayTest extends TypedArrayTest[Int, Uint8ClampedArray] {
+  def ofFn(items: Int*): Uint8ClampedArray = Uint8ClampedArray.of(items: _*)
+  def fromFn(iterable: js.Iterable[Int]): Uint8ClampedArray = Uint8ClampedArray.from(iterable)
+  def fromFn[E](iterable: js.Iterable[E])(fn: js.Function1[E, Int]): Uint8ClampedArray = Uint8ClampedArray.from(iterable, fn)
```

Comment:
The CI says that this line is too long.

Reply:
Fixed in https://github.com/scala-js/scala-js/commit/a7dc30d9c1e6a518e8c12615b41c51d680aa5f0a
(commits are squashed as requested always)